server/gameconnection.py

```diff
@@ -63,6 +63,21 @@ def __init__(self, loop: asyncio.BaseEventLoop,
 
         self.finished_sim = False
 
+        self.handlers = {
+            "Desync":               GameConnection.handle_desync,
+            "GameState":            GameConnection.handle_game_state,
+            "GameOption":           GameConnection.handle_game_option,
+            "GameMods":             GameConnection.handle_game_mods,
+            "PlayerOption":         GameConnection.handle_player_option,
+            "AIOption":             GameConnection.handle_ai_option,
+            "ClearSlot":            GameConnection.handle_clear_slot,
+            "GameResult":           GameConnection.handle_game_result,
+            "OperationComplete":    GameConnection.handle_operation_complete,
+            "JsonStats":            GameConnection.handle_json_stats,
+            "EnforceRating":        GameConnection.handle_enforce_rating,
+            "TeamkillReport":       GameConnection.handle_teamkill_report
```

Comment:
Because I want the function table to exist in a static context. I moved it to a better location after your review. 

Using the method via the class means that the function table can be moved to any file and then imported. Helps clean up the code.

Reply:
Half of those functions need self (self.game, etc...) how can it be static O_o?